### PR TITLE
fix: Tweak difftastic colors

### PR DESCRIPTION
**Pull request by gerhardol**
## Proposed changes

Use reverse colors also for difftastic bold colors, to improve visibility
By default difftastic use normal colors, just like diff.
This was modified to use reverse color for normal red/green, but difftastic uses bold for changed words.
https://github.com/Wilfred/difftastic/blob/master/src/display/style.rs#L313
https://github.com/Wilfred/difftastic/blob/master/src/display/style.rs#L368
https://github.com/Wilfred/difftastic/blob/master/src/display/style.rs#L384

Note that difftastic uses dim colors for "unchanged blocks" like when there are only whitespace changes.
This PR does not change that, see an example below of how that would look.
https://github.com/Wilfred/difftastic/blob/master/src/display/style.rs#L396

Darken some foreground colors to improve visibility, even if the difference between normal and bold is smaller.
Even if the colors are not directly used in difftastic or other diffs, they could by configured to used by users.
Also, the dimmed colors are partly used by difftastic for "non interesting", example below is changed just for whitespace.

## Screenshots 

difftastic1 difftastic2

Normal diff and difftastic default
difftastic3
difftastic0

resx1 resx2

The above diff with normal, then with dim background (not used)
resx3 resx4

color1 color2

## Test methodology 

manual

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: src/app/GitUI/Themes/invariant.css [hunk 1]
```diff
@@ -58,13 +58,13 @@
 .AnsiTerminalRedForeBold { color: #ff5e5e; }
 .AnsiTerminalRedBackBold { color: #ffa5a5; }
 .AnsiTerminalRedBackBold.colorblind { color: #4da6ff; }
-.AnsiTerminalGreenForeNormal { color: #1ca800; }
+.AnsiTerminalGreenForeNormal { color: #189100; }
 .AnsiTerminalGreenBackNormal { color: #c8ffc8; }
-.AnsiTerminalGreenForeBold { color: #2fd62f; }
+.AnsiTerminalGreenForeBold { color: #1ca800; }
```
Comment (mstv) [inline src/app/GitUI/Themes/invariant.css:63]:
```suggestion
.AnsiTerminalGreenForeBold { color: #21c800; }
```

File: src/app/GitUI/Themes/invariant.css [hunk 2]
```diff
@@ -58,13 +58,13 @@
...
 .AnsiTerminalGreenBackBold { color: #87ff87; }
 .AnsiTerminalYellowForeNormal { color: #c0a000; }
 .AnsiTerminalYellowBackNormal { color: #f2f293; }
-.AnsiTerminalYellowForeBold { color: #f2d543; }
+.AnsiTerminalYellowForeBold { color: #E2C73F; }
```
Comment (mstv) [inline src/app/GitUI/Themes/invariant.css:67]: nit
```suggestion
.AnsiTerminalYellowForeBold { color: #e2c73f; }
```

File: src/app/GitUI/Themes/invariant.css (hunk 1 above)
Comment (gerhardol) [reply src/app/GitUI/Themes/invariant.css]: thanks!

**Review by mstv — APPROVED** [review]
+1

**Comment by gerhardol** [comment]
A better example how dimmed colors (in comments) look. Dimmed are a little too close to normal, but the original color is so bright that this is hard to adjust.
Difftastic uses italics, that is not implemented in ICSharkCode highligting engine. (italics is implemented as dim)

image
image

image
image

**Pull request merged**